### PR TITLE
Cb 50/add message when user already exists

### PR DESCRIPTION
**Pull request by flying-pi**
Fix for the issue https://youtrack.raccoongang.com/issue/CB-50

### CODE REVIEW

File: lms/djangoapps/dashboard/tests/test_sysadmin.py [hunk 1]
```diff
@@ -363,3 +365,84 @@ def test_gitlog_courseteam_access(self):
...
+            'student_fullname': 'testser withsamename',
+            'student_password': 'edx',
+            'action': 'create_user',
+        }
+
+        self.client.post(reverse('sysadmin'), data=data)
+
+        try:
+            User.objects.get(username=data['student_uname'])
+        except:
```
Comment (idegtiarov) [inline lms/djangoapps/dashboard/tests/test_sysadmin.py]: add exception

File: lms/djangoapps/dashboard/tests/test_sysadmin.py [hunk 2]
```diff
@@ -363,3 +365,84 @@ def test_gitlog_courseteam_access(self):
...
+            'student_uname': 'test_user_with_same_name@example.com',
+            'student_fullname': 'testser withsamename',
+            'student_password': 'edx',
+            'action': 'create_user',
+        }
+
+        self.client.post(reverse('sysadmin'), data=data)
+
+        try:
+            User.objects.get(username=data['student_uname'])
```
Comment (idegtiarov) [inline lms/djangoapps/dashboard/tests/test_sysadmin.py]: add the comment about this try except block

File: lms/djangoapps/dashboard/tests/test_sysadmin.py [hunk 3]
```diff
@@ -363,3 +365,84 @@ def test_gitlog_courseteam_access(self):
...
+            create_result
+        )
+
+        create_result = users_api.create_user(**data)
+        self.assertEqual(
+            'This username already taken',
+            create_result
+        )
+
+        taken_email ='taken_email@example.com'
```
Comment (idegtiarov) [inline lms/djangoapps/dashboard/tests/test_sysadmin.py]: Please add comments and space after eq operator

File: lms/djangoapps/dashboard/tests/test_sysadmin.py [hunk 4]
```diff
@@ -363,3 +365,84 @@ def test_gitlog_courseteam_access(self):
...
+        User.objects.create_user(
+            username='one_more_user',
+            email=taken_email,
+        )
+        data = {
```
Comment (idegtiarov) [inline lms/djangoapps/dashboard/tests/test_sysadmin.py]: data could be updated instead of creating new object

**Comment by flying-pi** [comment]
start unit tests
start js tests

**Comment by hetmantsev** [comment]
start all tests

**Comment by flying-pi** [comment]
start all tests

**Comment by hetmantsev** [comment]
start all tests

**Comment by flying-pi** [comment]
start all tests

**Pull request merged**